VERSION

```diff
@@ -1 +1 @@
-1.3.5
+1.3.4
```

Comment:
Why the version was downgraded?

Reply:
Artem asked to set version to exact value 1.3.4